### PR TITLE
Translation files from IBM Translation Centers

### PR DESCRIPTION
**Pull request by sxshi**
Include the yaml and json files.

### CODE REVIEW

File: src/lang/fr.yaml [hunk 1]
```diff
@@ -1,81 +1,84 @@
...
+    
     # agent
     ## test: The next 2 appear when first starting the chat widget.
-    prompt: 'Écrire un message ...'
-    thinking: "L’agent réfléchit…"
+    prompt: 'Entrez un message...'
+    thinking: 'L''agent réfléchit...'
     ## test: This is shown after invoking 'list locations' and only getting a single entry in the list (2+ entries show text from the dialog).
-    anything_else: "Puis-je faire quelque chose d’autre pour vous aider?"
+    anything_else: 'Puis-je vous aider autrement ?'
```
Comment (jhpedemonte) [inline src/lang/fr.yaml]: Extra space before question mark

File: src/lang/fr.yaml [hunk 2]
```diff
@@ -1,81 +1,84 @@
...
     list_sep: ', '
 
     # errors
     ## test: Init IBMChat with an invalid `baseURL` to see the following.
-    reconnect: 'Tentative de reconnexion..'
-    basic_err: 'Je suis désolé, il y a un problème.'
-    loading_failure1: '<span class="IBMChat-loading-failure-message-text">Je ne peux pas compléter votre demande. Vous pouvez essayer une nouvelle demande ou <a class="IBMChat-loading-retry-link" data-retry="true" href="javascript:void(false);">recommencer la conversation.</a></span>'
-    loading_failure2: '<span class="IBMChat-loading-failure-message-text">Je ne peux pas compléter votre demande. Merci de <a class="IBMChat-loading-retry-link" data-retry="true" href="javascript:void(false);">recommencer la conversation.</a></span>'
+    reconnect: 'Tentative de reconnexion...'
+    basic_err: 'Je suis désolé ; je rencontre des difficultés.'
```
Comment (jhpedemonte) [inline src/lang/fr.yaml]: Extra space before `;`

File: src/lang/fr.yaml [hunk 3]
```diff
@@ -1,81 +1,84 @@
...
-    cc_code: 'Cryptogramme'
+    cc_name: 'Nom sur la carte'
+    cc_num: 'Numéro de carte de crédit'
+    cc_exp_mon: 'Mois de la date d''expiration'
+    cc_exp_year: 'Année de la date d''expiration'
+    cc_code: 'Code de sécurité'
     # cc_use_valid: Displayed as 'Please use a valid card. We accept the following: American Express, Mastercard, Visa', using `list_sep` to separate list items.
     ## test: Enter '1234' for the credit card number and submit.
-    cc_use_valid: 'Merci de fournir un numéro de carte valide. Nous acceptons: '
+    cc_use_valid: 'Veuillez utiliser une carte valide. Nous acceptons les cartes suivantes : '
```
Comment (jhpedemonte) [inline src/lang/fr.yaml]: Extra space before `:`

File: src/lang/fr.yaml [hunk 4]
```diff
@@ -1,81 +1,84 @@
...
     loc_hours_open_multiday: '<span class="{ns}-days-hours-day">{openDay, date, weekday_short}&ndash;{closeDay, date, weekday_short}:</span>&nbsp;<span class="{ns}-days-hours-hours">{open, time, short} &ndash; {close, time, short}</span>'
-    loc_hours_unknown: '<span class="{ns}-days-hours-day">{day, date, weekday_short}:</span>&nbsp;<span class="{ns}-days-hours-hours">Heures inconnues</span>'
+    loc_hours_unknown: '<span class="{ns}-days-hours-day">{day, date, weekday_short}:</span>&nbsp;<span class="{ns}-days-hours-hours">Horaires inconnus</span>'
     loc_hours_closed: '<span class="{ns}-days-hours-day">{day, date, weekday_short}:</span>&nbsp;<span class="{ns}-days-hours-hours">Fermé</span>'
     ## test: No locations added to 'store location' capability.
-    loc_nearby_none: 'Aucun résultat disponible.'
+    loc_nearby_none: 'Nous n''avons pas trouvé d''emplacements proches de vous.'
     ## test: Only one location added to 'store location' capability.
-    loc_nearby_single: 'Voici les informations pour ce site:'
+    loc_nearby_single: 'Voici les détails de cet emplacement :'
```
Comment (jhpedemonte) [inline src/lang/fr.yaml]: Extra space before `:`

File: src/lang/fr.yaml [hunk 5]
```diff
@@ -1,81 +1,84 @@
...
     ## test: More than one location added to 'store location' capability.
-    loc_nearby_list: 'Voici les sites à proximité:'
+    loc_nearby_list: 'Voici les emplacements que nous avons trouvé à votre proximité :'
```
Comment (jhpedemonte) [inline src/lang/fr.yaml]: Extra space before `:`

File: src/lang/pt-br.yaml [hunk 6]
```diff
@@ -1,53 +1,84 @@
...
+    loc_hours_closed: '<span class="{ns}-days-hours-day">{day, date, weekday_short}:</span>&nbsp;<span class="{ns}-days-hours-hours">Fechado</span>'
+    ## test: No locations added to 'store location' capability.
+    loc_nearby_none: 'Não foi possível encontrar localizações perto de você.'
+    ## test: Only one location added to 'store location' capability.
+    loc_nearby_single: 'Estes são os detalhes para esta localização:'
+    ## test: More than one location added to 'store location' capability.
+    loc_nearby_list: 'Estas são as localizações encontradas perto de você:'
+    # Used to generate a URL of form: https://maps.google.com/?q=store%20location
+    ## test: Click on address link for location details.
+    google_maps_url: 'https://maps.google.br/'
```
Comment (jhpedemonte) [inline src/lang/pt-br.yaml]: Not a valid site. Should be `https://maps.google.com.br`

**Comment by jhpedemonte** [comment]
In quick testing, only big issue I found was with pt-br maps link. For French, not sure if extra spaces were intentional or not. Will merge when issues fixed.

**Comment by sxshi** [comment]
Thank you, @jhpedemonte.  Our TVT starts from today, I will let the testers to fix it.

**Comment by ethanwinters** [comment]
@sxshi Javier is out on PTO. Where do we stand with this? Looks like we need you folks to fix merge conflicts.

**Comment by sxshi** [comment]
Closed this one, since the translation files were updated.

**Pull request closed without merging**